Doc/whatsnew/3.12.rst

```diff
@@ -816,7 +816,7 @@ Modules (see :pep:`594`):
 * :mod:`nntplib`
 * :mod:`ossaudiodev`
 * :mod:`pipes`
-* :mod:`sndhdr`
+* ``sndhdr``
```

Comment:
An alternative is to add an exclamation mark. This tells Sphinx not to look up the reference, but it still formats it consistently like a module.

```suggestion
* :mod:`!sndhdr`
```